scripts/diagnose.js

```diff
@@ -0,0 +1,40 @@
+const { spawnSync } = require("child_process")
+const fs = require("fs")
+
+const exec = (command, cwd) => {
+  const task = spawnSync(command, { shell: true, cwd })
+  if (task.status != 0) {
+    throw new Error(task.stderr.toString())
+  }
+  return task.stdout.toString()
+}
+
+const checkEnvVariables = () => {
+  // exec("touch .env.temp")
+  try {
+    exec("aws s3 cp s3://artsy-citadel/dev/.env.eigen .env.temp")
```

Comment:
should this be `.env.force`?

Reply:
I didn't meant to check that one in. That was a mistake.